reporter/src/test/kotlin/reporters/NoticeReporterTest.kt

```diff
@@ -32,13 +32,16 @@ import io.kotlintest.specs.WordSpec
 import java.io.File
 
 class NoticeReporterTest : WordSpec() {
+    companion object {
+        private fun readOrtResult(file: String) = File(file).readValue<OrtResult>()
+    }
+
     init {
         "Notices reporter" should {
             "generate the correct license notes" {
                 val expectedResultFile = File("src/test/assets/NPM-is-windows-1.0.2-expected-NOTICE")
                 val expectedText = expectedResultFile.readText()
-                val scanRecordFile = File("src/test/assets/NPM-is-windows-1.0.2-scan-result.json")
-                val ortResult = scanRecordFile.readValue<OrtResult>()
+                val ortResult = readOrtResult("src/test/assets/NPM-is-windows-1.0.2-scan-result.json")
```

Comment:
Hmm. I think I would have simply concatenated the original two lines without introducing a helper function, but ok.

Reply:
Actually, exactly what you do in the next commit.